meta-lmp-base/recipes-core/initrdscripts/initramfs-module-install-efi/init-install-efi.sh

```diff
@@ -239,58 +239,52 @@ mkdir -p /boot
 
 # Handling of the target root partition
 mount $rootfs /tgt_root
-mount -o rw,loop,noatime,nodiratime /run/media/$1/$2 /src_root
+mount -o rw,loop,noatime,nodiratime /run/media/${live_dev_name}3/rootfs.img /src_root
```

Comment:
is the `rootfs.img` pretended? it seams a typo

Reply:
It is always rootfs.img, which is why I changed to be explicit here.